docs/pages/setup/reference/moderated-sessions.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Moderated Sessions
+description: Moderated Sessions RBAC Reference
+h1: Moderated Sessions
+---
+
+# Moderated Sessions
+
+Moderated Sessions makes use of RBAC to control access to and within sessions.
+Here is a collection of details on configuring Moderated Sessions.
+
+## RBAC
+
+Moderated Sessions makes use of RBAC policies to allow for fine grained control over
+who can join a session and who are required to be present to start one.
+
+The system is based around **require policies** and allow policies. Require policies define a set of conditions
```

Comment:
```suggestion
The system is based around **require policies** and **allow policies**. 

Require policies define a set of conditions
```

Since we introduce both concepts in the same sentence, I think it looks a bit cleaner to separate it from the sentence that goes in depth on require policies.